### PR TITLE
Replace Azure with Actions

### PR DESCRIPTION
**Pull request by Daltz333**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        - name: Build Installer
+          uses: eskatos/gradle-command-action@v1
+          with:
+            gradle-version: 6.0.1
+            arguments: generateInstallers -PXmx3072m ${{ matrix.build-options }} -DsystemProp.org.gradle.internal.http.connectionTimeout=180000 -DsystemProp.org.gradle.internal.http.socketTimeout=180000 -PjenkinsBuild
+        - name: Archive Artifacts
+          uses: actions/upload-artifact@v2
+          with:
+            name: ${{ matrix.artifact-name }}-installer
+            path: build\outputs\
```
Comment (AustinShalit) [inline .github/workflows/ci.yml:46]:
```suggestion
            path: build/outputs/
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      steps:
+        - uses: actions/checkout@v2
+        - uses: actions/setup-dotnet@v1
+          with:
+            dotnet-version: 5.0.100-preview.8.20417.9
+        - uses: actions/setup-java@v1
+          with:
+            java-version: 11
+        - name: Build Installer
+          uses: eskatos/gradle-command-action@v1
```
Comment (AustinShalit) [inline .github/workflows/ci.yml:38]: Just run gradle directly here... no need to use a separate action.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      name: "Build - ${{ matrix.artifact-name }}"
+      runs-on: ${{ matrix.os }}
+      steps:
+        - uses: actions/checkout@v2
+        - uses: actions/setup-dotnet@v1
+          with:
+            dotnet-version: 5.0.100-preview.8.20417.9
+        - uses: actions/setup-java@v1
+          with:
+            java-version: 11
```
Comment (AustinShalit) [inline .github/workflows/ci.yml:36]: Make sure to specify arch here if we need 32b java.

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+          uses: eskatos/gradle-command-action@v1
+          with:
+            gradle-version: 6.0.1
+            arguments: generateInstallers -PXmx3072m ${{ matrix.build-options }} -DsystemProp.org.gradle.internal.http.connectionTimeout=180000 -DsystemProp.org.gradle.internal.http.socketTimeout=180000 -PjenkinsBuild
+        - name: Archive Artifacts
+          uses: actions/upload-artifact@v2
+          with:
+            name: ${{ matrix.artifact-name }}-installer
+            path: build\outputs\
+
```
Comment (AustinShalit) [inline .github/workflows/ci.yml:47]: eol @ eof

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+            java-version: 11
+        - name: Build Installer
+          uses: eskatos/gradle-command-action@v1
+          with:
+            gradle-version: 6.0.1
+            arguments: generateInstallers -PXmx3072m ${{ matrix.build-options }} -DsystemProp.org.gradle.internal.http.connectionTimeout=180000 -DsystemProp.org.gradle.internal.http.socketTimeout=180000 -PjenkinsBuild
+        - name: Archive Artifacts
+          uses: actions/upload-artifact@v2
+          with:
+            name: ${{ matrix.artifact-name }}-installer
```
Comment (AustinShalit) [inline .github/workflows/ci.yml:45]:
```suggestion
            name: ${{ matrix.artifact-name }}
```

**Review by AustinShalit — CHANGES_REQUESTED** [review]

**Comment by ThadHouse** [comment]
Can you pass a wildcard for the dotnet version? I'm going to keep updating it. 

Also, I'm not fully in line with this one switching. This job fails a lot, and rerunning only the failed builds is basically the only way to actually get a full build.

**Comment by AustinShalit** [comment]
Wildcard dotnet version support is pending this: https://github.com/actions/setup-dotnet/pull/110

**Review by AustinShalit — APPROVED** [review]

**Pull request merged**